packages/context-menu/package.json

```diff
@@ -23,7 +23,13 @@
     "lit.d.ts",
     "lit.js",
     "src",
```

Comment:
Could use a [wildcard](https://github.com/vaadin/web-components/blob/0271523d93fe5df0425ff64206886614f3c6f401/packages/button/package.json#L29-L30) rather than listing the individual files

Reply:
The current revision doesn't exclude the files under `src`. This is what I meant:

```json
  "files": [
    "lit.d.ts",
    "lit.js",
    "src",
    "theme",
    "vaadin-*.d.ts",
    "vaadin-*.js",
    "web-types.json",
    "web-types.lit.json",
    "!vaadin-lit-*.d.ts",
    "!vaadin-lit-*.js"
  ],
```

Reply:
Thanks, updated.